sample_project/sample_app/tests/tests_tasks/tests_routine_executor_tasks.py

```diff
@@ -0,0 +1,191 @@
+from datetime import datetime, UTC
+from unittest.mock import patch
+
+from django.test import TestCase
+
+from django_cloud_tasks.tasks import RoutineExecutorTask, TaskMetadata
+from django_cloud_tasks.tests import factories, tests_base
+from django_cloud_tasks.tests.tests_base import EagerTasksMixin
+from sample_app import tasks
+from sample_app.tests.tests_base_tasks import patch_cache_lock
+
+
+class RoutineExecutorTaskTest(EagerTasksMixin, TestCase):
+    _mock_lock = None
+
+    def setUp(self):
+        super().setUp()
+
+        self.mock_lock = patch_cache_lock()
+        self.mock_lock.start()
+        self.addCleanup(self.mock_lock.stop)
+
+    def assert_routine_lock(self, routine_id: int):
+        self.mock_lock.assert_called_with(
+            key=f"lock-RoutineExecutorTask-{routine_id}",
+            timeout=60,
+            blocking_timeout=5,
+        )
+
+    def tests_dont_process_completed_routine(self):
```

Comment:
```suggestion
    def test_dont_process_completed_routine(self):
```
Same for other tests.

Reply:
fixed, tks!